### ISSUE TITLE
Storyboarder gets stuck on "Initializing" project

### ISSUE THREAD

**Issue by sanr12** [wonderunit/storyboarder#1435]
Hey all,

I'm trying to test out Storyboarder but it gets stuck after I create a new project. It stays on the "initializing" screen and never actually opens. 

I'm running a OS X 10.9.5 from mid 2012. Any way to fix the problem?

![screen shot 2018-11-06 at 10 34 48 pm](https://user-images.githubusercontent.com/44811600/48075363-b42f7600-e215-11e8-81b8-32d269badc76.png)

**Comment by audionerd** [comment]
Can you try opening Developer Tools when this happens and let us know what you see in the console?

**Comment by sanr12** [comment]
Is this what you mean? The console seems empty. 


![screen shot 2018-11-06 at 11 16 05 pm](https://user-images.githubusercontent.com/44811600/48077479-f955a700-e219-11e8-93da-13a1a0ea6f97.png)

**Comment by sanr12** [comment]
I tried opening developer tools on the main menu as well. Here are the messages I got.

![screen shot 2018-11-06 at 11 31 53 pm](https://user-images.githubusercontent.com/44811600/48078666-7da92980-e21c-11e8-911c-f8a4d02dbaf7.png)
![screen shot 2018-11-06 at 11 32 23 pm](https://user-images.githubusercontent.com/44811600/48078743-99accb00-e21c-11e8-8147-fc070cfda0ef.png)

And here is developer tools on the screen that the program gets stuck on.

![screen shot 2018-11-06 at 11 33 11 pm](https://user-images.githubusercontent.com/44811600/48078796-aaf5d780-e21c-11e8-8e7f-5c44faa83d65.png)

**Comment by sanr12** [comment]
The developer tools box quickly disappears after it shows up though.

**Comment by audionerd** [comment]
OK, there's a way to debug this but it's a little tricky ... Please bear with me!

When you get the Developer Tools open on this window:

![image](https://user-images.githubusercontent.com/23459/48093001-c7215680-e1d3-11e8-8ac8-4c582f5cb1bb.png)

Try pasting the following command text there (click next to the little blue arrow prompt, then paste) and press enter:

`require('electron').remote.BrowserWindow.getAllWindows().forEach(w => w.show())`

If it works, you'll see the partially loaded project window appear.

Then, open Developer Tools on that project window, and let us know what you see in the logs.

**Comment by sanr12** [comment]
Hey,

I've been trying for the past half hour, but it seems the window closes too quickly to receive any response (it only opens for under a second). I get of all the proper commands before it closes but it simply seems unresponsive. Is this the only way to troubleshoot the program? I can keep trying later if so.

**Comment by audionerd** [comment]
You can try the same command from any window you are able to open. So if that closes, maybe try the Key Commands window, and run Developer Tools, and type it there?

**Comment by sanr12** [comment]
I still can't get it to work but I do have a few updates. On the main Key Command window I was able to keep Developer tools open for a few seconds by clicking on the Network tab. But as soon as I clicked something else the developer tools window disappeared and I can't seem to duplicate the process.

Also some new messages have appeared under the console tab. I've attached all my screenshots.
![screen shot 2018-11-10 at 9 16 12 pm](https://user-images.githubusercontent.com/44811600/48302494-32e72480-e530-11e8-908c-0ca9ea27c168.png)
![screen shot 2018-11-10 at 8 58 33 pm](https://user-images.githubusercontent.com/44811600/48302498-37abd880-e530-11e8-8df3-3cc4e9ddcfe2.png)
![screen shot 2018-11-10 at 9 11 19 pm](https://user-images.githubusercontent.com/44811600/48302501-409caa00-e530-11e8-8432-93948a084761.png)
